packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -564,37 +578,93 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
         }
     }
 
-    private tooltip(x: number, y: number): void {
+    private closestPoint(array: any, point: any, xRatio: number, yRatio: number, chartHeight: number): number | undefined {
+        const hypos: any[] = [];
+        // Convert x and y values to screen pixels
+        const pointX = point.x * xRatio;
+        const pointY: number = chartHeight - point.y;
+
+        array.forEach((ar: any) => {
+            const x = ar.x * xRatio;
+            const y = (ar.y - this.state.allMin) * yRatio;
+            const hypotenuse = Math.sqrt(Math.pow(Math.abs(pointX - x), 2) + Math.pow(Math.abs(pointY - y), 2));
+            hypos.push(hypotenuse);
+        });
+
+        const min = Math.min(...hypos);
+
+        if (min < 20) {
```

Comment:
if you remove the squareroots, min < 400

Reply:
Yes, thank you for the help there.